ida-customizations/.vuepress/config.js

```diff
@@ -295,7 +299,7 @@ module.exports = {
                   path: "/hands-on-exercise/1-ignite-cli/8-game-winner.html",
                 },
                 {
-                  title: "Week 2 Exercise",
+                  title: "Coding Exercise 1",
```

Comment:
```suggestion
                  title: "Exam Coding Exercise 1",
```

I am thinking to make it like so.